.github/workflows/home.yml

```diff
@@ -55,3 +55,39 @@ jobs:
           status: ${{ job.status }}
           fields: job,message,ref,eventName,author,took
           author_name: Home app
+
+  publish-dogfood-home:
+    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+    needs: build
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14.17'
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
```

Comment:
This caching approach turned out not to give much profit, I recommend something like this: https://github.com/expo/expo/blob/ec71f441da6f3975a424aec03d8d4f2efc6a26c2/.github/workflows/sdk.yml#L41-L54

Reply:
Good to know! To limit the scope of this PR, I'm going to keep this the same as the job above and change both in a follow-up task: https://linear.app/expo/issue/ENG-1748/investigate-improving-home-workflow-caching